CarPlay documentation (#923)
home-assistant/companion.home-assistant: docs/carplay/carplay.md

---
title: "Overview"
id: "carplay"
---

![iOS](/assets/iOS.svg)

> Home Assistant has started to offer an CarPlay experience.  This will allow you to interact with various entities safely while driving your vehicle.

Reviewer: ```suggestion
Home Assistant offers a CarPlay experience. This will allow you to interact with various entities safely while driving your vehicle.
```

We should keep the language generic and just document what can be done rather than promoting as new. A blog post on the main site can present that this is new.

### Setup

> In order to use this integration you will need an iPhone as well as a vehicles head unit that supports CarPlay.  Once you are signed in with your iPhone, you should be all set to use the Home Assistant icon on the CarPlay home screen.

Reviewer: ```suggestion
In order to use this integration you will need an iPhone as well as a vehicle with a head unit that supports CarPlay. Once you are signed in with your iPhone, you should be all set to use the Home Assistant icon on the CarPlay home screen.
```
